tests/test_ops.py

```diff
@@ -1395,14 +1412,29 @@ def test_mathtransform():
+        #{"opName": "zeros_like", "outName": "zeros_like/rank2_float64", "varShapes": [[3, 12]], "varTypes": ["float64"], "varInit": ["uniform"]},
+        #{"opName": "zeros_like", "outName": "zeros_like/rank2_int32", "varShapes": [[3, 10]], "varTypes": ["int32"], "varInit": ["uniform_int10"]},
+        #{"opName": "zeros_like", "outName": "zeros_like/rank2_int64", "varShapes": [[3, 10]], "varTypes": ["int64"], "varInit": ["uniform_int10"]},
 
         # {"opName": "cast", "outName": "emptyArrayTests/cast/rank2_shape2-0", "varShapes":[[2,0]], "varTypes":["float32"], "varInit":["empty", "empty"], "dtype":tf.int32},
         # {"opName": "cast", "outName": "emptyArrayTests/cast/rank1_shape0", "varShapes":[[0]], "varTypes":["int32"], "varInit":["empty", "empty"], "dtype":tf.int64},
 
 
-        # {"opName": "abs", "outName": "emptyArrayTests/abs/rank1", "varShapes":[[0]], "varTypes":["int32"], "varInit":["empty"], "dtype":tf.int32},
-        # {"opName": "abs", "outName": "emptyArrayTests/abs/rank2", "varShapes":[[0,2]], "varTypes":["float64"], "varInit":["empty"], "dtype":tf.float64},
-        #
+        #{"opName": "abs", "outName": "emptyArrayTests/abs/rank1", "varShapes":[[0]], "varTypes":["int32"], "varInit":["empty"], "dtype":tf.int32},
+        #{"opName": "abs", "outName": "emptyArrayTests/abs/rank2", "varShapes":[[0,2]], "varTypes":["float64"], "varInit":["empty"], "dtype":tf.float64},
+        #{"opName": "abs", "outName": "abs/rank2_float64", "varShapes": [[1, 10]], "varTypes": ["float64"], "varInit": ["uniform"], "dtype": tf.float64},
+        #{"opName": "abs", "outName": "abs/rank2_float32", "varShapes": [[1, 10]], "varTypes": ["float32"], "varInit": ["uniform"], "dtype": tf.float32},
+        #{"opName": "abs", "outName": "abs/rank2_float16", "varShapes": [[1, 10]], "varTypes": ["float16"], "varInit": ["uniform"], "dtype": tf.float16},
+        #{"opName": "abs", "outName": "abs/rank2_int32", "varShapes": [[1, 10]], "varTypes": ["int32"], "varInit": ["uniform_int10"], "dtype": tf.int32},
+        #{"opName": "abs", "outName": "abs/rank2_int64", "varShapes": [[1, 10]], "varTypes": ["int64"], "varInit": ["uniform_int10"],"dtype": tf.int64},
+        #{"opName": "abs", "outName": "abs/rank3_float64", "varShapes": [[1, 10, 20]], "varTypes": ["float64"], "varInit": ["uniform"], "dtype": tf.float64},
+        #{"opName": "abs", "outName": "abs/rank3_float32", "varShapes": [[1, 10, 20]], "varTypes": ["float32"], "varInit": ["uniform"], "dtype": tf.float32},
+        #{"opName": "abs", "outName": "abs/rank3_float16", "varShapes": [[1, 10, 20]], "varTypes": ["float16"], "varInit": ["uniform"], "dtype": tf.float16},
+        #{"opName": "abs", "outName": "abs/rank3_int32", "varShapes": [[1, 10, 20]], "varTypes": ["int32"], "varInit": ["uniform_int10"], "dtype": tf.int32},
+        #{"opName": "abs", "outName": "abs/rank3_int64", "varShapes": [[1, 10, 20]], "varTypes": ["int64"], "varInit": ["uniform_int10"], "dtype": tf.int64},
```

Comment:
abs op: this is absolute value operation.
"uniform" and "uniform_intX" initializations are all positive.
Initializations must include some negative and some positive values to properly test the op.

Reply:
Added random normal inits with zero mediana.